### DISCUSSION TITLE
How to apply styles to the new NextJS Image component?

### DISCUSSION THREAD

**Discussion by oahmaro** [chakra-ui/chakra-ui#2558] [Q&A]
Is there a guide on how to use NextJs Image with chakra, and have all BoxProps so that we can control styling on image?

**Comment by musyyy** [comment]
Maybe this gist can help you:https://gist.github.com/TheThirdRace/0f439acef8d9cb6bf5d7e69c54704086

Source: https://discord.com/channels/660863154703695893/660865137137156146/784892422160777227
